Fixes for managing content views chap (#1637)
theforeman/foreman-documentation: guides/common/modules/proc_promoting-a-content-view.adoc

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and select the Content View that you want to promote.

> . Select the version that you want to promote, click the vertical ellipsis icon and click *Promote*.

Reviewer: ```suggestion
. Select the version that you want to promote, click the vertical ellipsis icon, and click *Promote*.
```

> . Select the environment where you want to promote the Content View, and click *Promote*.

Reviewer: ```suggestion
. Select the environment where you want to promote the Content View and click *Promote*.
```

Can you reword "where"? Sounds a bit off to me.

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and select the Content View that you want to promote.

> . Click the *Versions* tab for the Content View.

Reviewer 2: ```suggestion
```

> . Select the version that you want to promote and click the *More Options* icon > *Promote*.

Reviewer 2: ```suggestion
. On the *Versions* tab, select the version that you want to promote and click on the vertical ellipsis on the right side of the screen.
. Click *Promote*.
```

Author: ```suggestion
. Select the version that you want to promote, click the vertical ellipsis icon and click *Promote*.
```
